authorizer-app-backend/src/main/resources/db/changelog/changes/00000000000003_update_schema.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet author="pauline" id="addCreatedAt">
+        <addColumn tableName="rest_source_user">
+            <column name="created_at" type="timestamp" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="true" />
```

Comment:
What happens to users which don't have computed values (prior registered users). I know this will probably never happen (since we will deploy after this is merged) but would be good to handle the case. I think those should have null values. AFAIR this is specified by the `value=NULL` property which will set existing rows to this value for the new column.

Reply:
I was thinking of making the default value `defaultValueComputed="CURRENT_TIMESTAMP"`?

Reply:
But that will only take effect on newly created users. It will not update the column values existing users (if any already present). Also even if it does, using the current timestamp for existing users create date would be incorrect since they would have authorised in the past. If it is null the external system can handle this case.

Reply:
Oh okay I see. I thought that the external system needed an actual value. I'll make the changes.

Reply:
hi Sorry, i think there was a misunderstanding. The new records should have the computed current timestamp (so keeping `defaultValueComputed="CURRENT_TIMESTAMP"` as it is) but also adding value=null for any existing records. This will only update the existing records without affecting the default value of the column.

Reply:
Ah, my bad I've made the changes.